pandas/plotting/_core.py

```diff
@@ -653,6 +653,16 @@ class PlotAccessor(PandasObject):
         Set the x limits of the current axes.
     ylim : 2-tuple/list
         Set the y limits of the current axes.
+    xlabel : label, default None
```

Comment:
```suggestion
    xlabel : label, optional
```

We usually use `default None` when the `None` is being used (imagine you have something like `fill_with=None`). When `None` means that the value is simply not provided, and the parameter won't be use, we use `optional`. A bit complex, but we've been somehow consistent with that.

Reply:
yep, thats fair! changed